src/kOS/Function/Math.cs

```diff
@@ -14,7 +15,7 @@ public override void Execute(SharedObjects shared)
             double argument = GetDouble(PopValueAssert(shared));
             AssertArgBottomAndConsume(shared);
             double result = Math.Abs(argument);
-            ReturnValue = result;
+            ReturnValue = Structure.FromPrimitive(result);
```

Comment:
Oh that wasn't aimed at you.  It was aimed at an edit that Resharper kept doing to my PRs when erendrake ran it that irked me.  Choosing whether an initializers is explicitly mentioned or not might not matter to the compiler, but it does matter to the human reading the code.  It communicates the difference between "initial value = The fact that it will start at zero is irrelevant" versus "initial value = my algorithm explicitly wanted it to be zero".  When my careful decision to mention some initializers explicitly when they were relevant to the algorithm kept getting squashed and removed under the false label of "redundant" by Resharper, it annoyed me.  Unless the compiler is really really dumb, writing something like `int x = 0;` isn't going to end up redundantly initializing it to zero twice.  It's going to notice there is an explicit initializer and use that *instead of* the implicit one, not *in addition to*, which is what the term 'redundant' would imply.